src/VisualStudio/Core/Test/ProjectSystemShim/VisualStudioProjectTests/AnalyzerReferenceTests.vb

```diff
@@ -204,5 +204,36 @@ Namespace Microsoft.VisualStudio.LanguageServices.UnitTests.ProjectSystemShim
                 AssertEx.Equal({path1, path2}, environment.Workspace.CurrentSolution.Projects.Single().AnalyzerReferences.Select(Function(r) r.FullPath))
             End Using
         End Function
+
+        <WpfFact>
+        Public Async Function CodeStyleAnalyzers_FromSdk() As Task
+            Using environment = New TestEnvironment()
+                Dim providerFactory = DirectCast(environment.ExportProvider.GetExportedValue(Of IVisualStudioDiagnosticAnalyzerProviderFactory), MockVisualStudioDiagnosticAnalyzerProviderFactory)
+                providerFactory.Extensions =
+                {
+                     ({
+                        Path.Combine(TempRoot.Root, "File.dll")
+                     },
+                     "AnotherExtension")
+                }
```

Comment:
Should we just remove this bit since it's not relevant to the test at hand? I imagine this has gotten copy/pastede from prior tests and didn't matter here. It'll also make the assert an Assert.Empty( )which will be a bit clearer.

Reply:
Removed!